lib/versions_repository.py

```diff
@@ -25,3 +25,17 @@ def setup_versions_repository(config):
         raise
 
     return versions_repo
+
+def read_version_and_milestone(versions_repo):
+    """
+    Read current version and milestone (alpha or beta) from VERSION file
+    """
+    version_path =  os.path.join(versions_repo.name, 'VERSION')
+    version_milestone_format = ""
+    version_milestone = ""
+    with open(version_path, 'r') as file_version:
+        version_milestone_format = file_version.readline()
```

Comment:
you could use _ instead of version_milestone_format or add a ignore directive to avoid a Pylint unused variable warning